_infra/Chart.yaml

```
@@ -1,5 +1,5 @@
 apiVersion: v1
 description: A golang short program that monitor certificates
 name: tlsmonitor
-version: 1.3.4
+version: 1.3.5
```

It looks like I forgot to bump the version here last time... It should be 1.3.6
```suggestion
version: 1.3.6
```